test/wdio.conf.js

```diff
@@ -11,16 +11,16 @@ exports.config = {
     {
       browserName: 'chrome',
 
-      chromeOptions: {
+      'goog:chromeOptions': {
         args: ['--headless', '--incognito'],
       },
     },
-    {
-      browserName: 'firefox',
-      'moz:firefoxOptions': {
-        args: ['--headless'],
-      },
-    },
+    // {
```

Comment:
Why is this commented out? Don't the tests pass on Firefox?

Reply:
@Ad1B3nAr0ya What about Firefox?

Reply:
some of the WDIO functions such as setWindowSize and scrollIntoView don't work on FF we need to see how we provide alternative in JS and if it should be part of our WD wrapper. 
TBD

Reply:
for setWindowSize try https://webdriver.io/docs/api/webdriver.html#setwindowrect